plugins/woocommerce/src/Admin/API/Options.php

```diff
@@ -235,6 +235,7 @@ public static function get_default_option_permissions() {
 			'woocommerce_show_lys_tour',
 			'woocommerce_remote_variant_assignment',
 			'woocommerce_gateway_order',
+			'woocommerce_woopayments_nox_profile', // This is temporary.
```

Comment:
Is this will stay temporary, or is it used when this is launched too?

Reply:
The thing is that I don't see a replacement for this logic to be able to allow folks to use the WCA options feature to edit options. This addition has nothing to do with the NOX functionality since we don't edit this option in-app, only through the backend.